ding2.install

```diff
@@ -564,3 +564,10 @@ function ding2_update_7045() {
 function ding2_update_7046() {
   ding2_translation_update();
 }
+
+/**
+ * Enable ding_variables module.
+ */
+function ding2_update_7047() {
+  module_enable(array('ding_variables'));
```

Comment:
This probably only needs to be enabled for libraries using `opensearch` - especially if `opensearch` is added as a dependency.